spec/services/converters/turtle_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Converters::Turtle do
+  describe ".convert" do
+    let(:graph) { result.fetch(:@graph) }
+    let(:result) { described_class.convert(file) }
+
+    let(:file) do
+      Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", filename))
+    end
+
+    context "N-Triples" do
+      let(:filename) { "ASN_D2695955.nt" }
+
+      it "converts N-Triples to JSON-LD" do
+        expect(result.keys).to eq(%i[@graph])
+
+        expect(graph.map {|r| r["@type"] }.uniq.compact).to match(
+          [
+            "http://purl.org/ASN/schema/core/Statement",
```

Comment:
Why the types of nodes should only be those? why to make the test so tied to the tested file?